### ISSUE TITLE
Bug related to clipboard in xvnc - affects this image

### ISSUE THREAD

**Issue by OriMankali** [jlesage/docker-baseimage-gui#100]
Hey @jlesage 

TL;DR: this is a kind request to release an official build docker-baseimage-gui and docker-firefox to include a bug fix.

Longer version:
I have encountered a bug which is related to xvnc process (tigervnc), the symptom is an assertion due to incorrect state and looks like that:

[xvnc        ] Thu Apr 13 07:37:17 2023
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ] Assertion failed: state_ >= RFBSTATE_QUERYING (/tmp/tigervnc/common/rfb/SConnection.cxx: accessCheck: 350)
[xvnc        ] (EE)
[xvnc        ] Fatal server error:
[xvnc        ] (EE) Caught signal 6 (Aborted). Server aborting
[xvnc        ] (EE)
[openbox     ] X connection to :0 broken (explicit kill or server shutdown).
[supervisor  ] service 'xvnc' exited (got signal SIGABRT).
[supervisor  ] service 'openbox' exited (with status 1).
[supervisor  ] service 'app' exited (with status 1).
[supervisor  ] service 'app' exited, shutting down...
[supervisor  ] stopping service 'nginx'...
[supervisor  ] service 'nginx' exited (with status 0).
[supervisor  ] sending SIGTERM to all processes...
[finish      ] executing container finish scripts...
[finish      ] all container finish scripts executed.

As you can see, this is causing the container to crash.
While debugging it, I have noticed that this bug was already fixed by the maintainer of tigervnc here:
https://github.com/TigerVNC/tigervnc/pull/1601

However - this fix is not included in the latest builds of docker-baseimage-gui and the derived images (I'm using docker-firefox, but I guess it can affect others).

I have verified the fix by building the image locally (which builds the latest version of tigervnc) - and confirmed that the bug was indeed fixed.

Can you please release an official build docker-baseimage-gui and docker-firefox?

Many thanks in advance,
Ori

**Comment by jlesage** [comment]
Sure, I currently preparing a new release to include the latest version of xvnc.

**Comment by jlesage** [comment]
A new image as already been released.
